docs/user-documentation/glossary.md

```diff
@@ -0,0 +1,79 @@
+The following glossary of terms addresses an Islandora 8 context. When comparing new Islandora and Fedora to older versions it may also be helpful to reference [the Islandora 7 Glossary](https://wiki.duraspace.org/display/ISLANDORA/APPENDIX+E+-+Glossary).
+ 
+## CLAW
+ CLAW (CLAW Linked Asset WebDataFrameWork) was the production code name for the software released in June, 2019 as _Islandora 8_
+ 
+## Collection
+Collections are a way of grouping related objects together in Islandora. They function much like directories on a computer; a collection can “contain” any number of related resource nodes and sub-collections. 
+ 
+## Drupal 
+Drupal is an open source content management system (CMS) written in PHP. Known for being extremely flexible and extensible, Drupal is supported by a community of over 630,000 users and developers. Drupal sites can be customized and themed in a wide variety of ways.
+ 
+Drupal functions as the _front-end_ of Islandora.
+ 
+## Drupal Core
+The files, themes, profiles, and modules included with the standard project software download.
+ 
+## Drupal Roles
+Roles are a way of assigning specific permissions to a group of users. Any user assigned to a role will have the same permissions as all other users assigned to that role. This allows you to control which users have permission to view, edit, or delete content in both Drupal and the Fedora repository.
+ 
+## Entity 
+A Drupal term for an item of either content or configuration data, although in common usage, the term often refers to content entities. Examples include content items, custom blocks, taxonomy terms, and definitions of content types; the first three are content entities, and the last is a configuration entity. 
+ 
+## Fedora (Repository Software)
+Fedora is a digital asset management architecture upon which institutional repositories, digital archives, and digital library systems might be built. Fedora is the underlying architecture for a digital repository, and is not a complete management, indexing, discovery, and delivery application. 
+ 
+The Fedora repository functions as the standard _smart storage_ for Islandora.
+ 
+## Field
+Data of a certain type that is attached to a content entity. For instance, on a Resource Node content type, you might have fields for a title, description, display hints, subjects, and other metadata. 
+ 
+## Flysystem
+Flysystem is a filesystem abstraction library for PHP. Islandora uses Flystsem to swap about different backend filesystem applications, and comes with a custom Flystsem adapter for Fedora.
```

Comment:
"filesystem applications, and comes with a custom Flystsem adapter for Fedora." should probably be "filesystem applications. Islandora provides a custom Flystsem adapter for Fedora."